frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -35,20 +35,20 @@ interface TrendPeople {
     people: PersonType[]
     breakdown_value?: string
     count: number
-    day: string | number
+    day?: string | number
     next?: string
     label: string
-    action: ActionFilter
+    action: ActionFilter | string
     loadingMore?: boolean
 }
 
 interface PeopleParamType {
-    action: ActionFilter
+    action: ActionFilter | string
```

Comment:
This looks like something once implemented, is hard to take back. 

Why do we need to water down the type to a string? Is this something that was just badly typed before, or is passing this as a string a new thing? Ideally it would be great to have as many explicit types as possible.

Reply:
It passes a string here :( https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/trends/viz/ActionsLineGraph.tsx#L49

Reply:
It's an old implementation and I've just been trying to type things as much as possible, it's probably out of scope in this PR for me to fix what was originally sent in those methods

Reply:
Is that the only place this string is passed? 
What about having the type as `ActionFilter | 'session'`? 
This would make it clearer that it's some temporary situation that should be refactored... and also make the `typeof action !== 'string' && action.id)...` code a bit simpler.

Reply:
And regarding "it's probably out of scope in this PR for me to fix what was originally sent in those methods", well, it's one line in one file that's impacting the quality of the new code, so it can definitely fall "in scope" if we want to.

Reply:
Yeah that's the only place that string is being called, I can leave a comment in that method since that's the only place it passes in `session`, the other loadPeople methods don't do that, so it might be less confusing when going back to refactor? I tried to pass in `ActionFilter | 'session'` for the type but it results in the same problem 
<img width="570" alt="Screen Shot 2021-06-29 at 6 05 58 PM" src="https://user-images.githubusercontent.com/25164963/123873566-a8cc7400-d904-11eb-90e7-02fd6ec8b56b.png">

Reply:
update: replaced the type and i'm checking here for `action !== 'session'` now 😄